bool isMsgFromAuto
int16 autoID         
int16 direction     
bool isCrossed       
int16[] avaibleDirections    
int16 length         
int16 nedxtCrossID     
int16 previousAutoID   
